app/models/agents/twitter_search_agent.rb

```diff
@@ -0,0 +1,97 @@
+module Agents
+  class TwitterSearchAgent < Agent
+    include TwitterConcern
+
+    cannot_receive_events!
+
+    description <<-MD
+      The Twitter Search Agent emits the results of a specified search.
+
+      #{twitter_dependencies_missing if dependencies_missing?}
+
+      To be able to use this Agent you need to authenticate with Twitter in the [Services](/services) section first.
+
+      You must provide the desired `search`.
+      
+      Set `result_type` (String) — Specifies what type of search results you would prefer to receive. Options are "mixed", "recent", and "popular". (default: `mixed`)
```

Comment:
Maybe you could add a link to the twitter docs where the different types are explained?